rlp/main.cpp

```diff
@@ -256,14 +257,20 @@ int main(int argc, char** argv)
         prefs.escapeAll = true;
 
     bytes in;
-    if (inputFile == "--")
-        for (int i = cin.get(); i != -1; i = cin.get())
-            in.push_back((byte)i);
-    else if (boost::filesystem::is_regular_file(inputFile))
-        in = contents(inputFile);
-    else
-        in = asBytes(inputFile);
-
+    try
```

Comment:
Don't make try-catch block wider than it needs to be - if you intend to handle errors from `is_regular_file` only (maybe together with `contents()`) - wrap only those functions in try-catch